File: Jumping Number (Special Numbers Series #4)/Jumping Number (Special Numbers Series #4).dart
String jumpingNumber(int n) {
  String n_s = n.toString();
  for(var i =0;i<n_s.length-1;i++){
    if((int.parse(n_s.substring(i,i+1))-int.parse(n_s.substring(i+1,i+2))).abs()!=1){
      return "Not!!";
    }
  }
  return "Jumping!!";
}

File: Jumping Number (Special Numbers Series #4)/Jumping Number (Special Numbers Series #4).swift
func jumpingNumber(_ number: Int) -> String {
    let digits = String(number).compactMap { $0.wholeNumberValue }
    
    for i in 0..<digits.count - 1 {
        if abs(digits[i + 1] - digits[i]) != 1 {
            return "Not!!"
        }
    }
    return "Jumping!!"
}
